gymnax/environments/misc/rooms.py

```diff
@@ -105,7 +105,7 @@ def step_env(
         new_pos = jax.lax.select(in_map, p, state.pos)
         reward = jnp.logical_and(
             new_pos[0] == state.goal[0], new_pos[1] == state.goal[1]
-        )
+        ) * 1.0
```

Comment:
nit: looks good. I was wondering why you didn't decide to use `jax.numpy.astype()` for this?